configs/example.yaml

```diff
@@ -0,0 +1,39 @@
+facility_name: SUBARU
```

Comment:
There's some evidence that "Subaru" may be the preferred spelling, with some existing use in IVOA services.  See, e.g., http://svo2.cab.inta-csic.es/theory/fps/index.php?mode=browse&gname=Subaru&gname2=HSC&asttype=

Also we need `instrument_name: HSC` here.